galley/pkg/source/kube/builtin/source_test.go

```diff
@@ -60,6 +62,10 @@ var (
 )
 
 func TestNewWithUnknownSpecShouldError(t *testing.T) {
+	// Set the log level to debug for codecov.
+	prevLevel := setDebugLogLevel()
```

Comment:
Can we just simply set the debug log level as part of init and not worry about resetting it? Would make the test code cleaner.

Reply:
I considered that, but I'd rather not interfere with other tests.